tools/cell_ranger/Dockerfile

```diff
@@ -42,15 +66,34 @@ ENV PATH="/root/.cargo/bin:$PATH"
 # install correct version of rustup
 RUN rustup install 1.28.0 && rustup default 1.28.0
 
+# Download precompiled martian and unpack
+RUN wget https://github.com/martian-lang/martian/releases/download/v3.2.0/martian-v3.2.0-linux-x86_64.tar.gz \
+  && tar -xf martian-v3.2.0-linux-x86_64.tar.gz && rm martian-v3.2.0-linux-x86_64.tar.gz
+
+# Install STAR aligner
+RUN wget https://github.com/alexdobin/STAR/archive/2.5.1b.tar.gz \
+ && tar -xf 2.5.1b.tar.gz \
+ && rm 2.5.1b.tar.gz \
+ && cd STAR-2.5.1b \
+ && make \
+ && cp bin/Linux_x86_64/STAR /usr/bin \
+ && cd .. \
+ && rm -rf STAR-2.5.1b
+
+# Install tsne
+RUN git clone https://github.com/wpoehlm/tsne.git \
+ && cd tsne \
+ && make install \
+ && cd .. \
+ && rm -rf tsne
+
 # clone cellranger sourcecode repo, cd into this dir, and install cellranger
-RUN git clone https://github.com/10XGenomics/cellranger \
+RUN git clone https://github.com/wpoehlm/cellranger.git \
```

Comment:
I might get the 3.0.2 release of this (git checkout the tag). You could use the variable above from your `software.version` label to manage that? Just on the off chance that you pull to master on this repository and something changes.

Reply:
I added git checkout commands with commit sha's, does that look alright or do you think it would be better to use the tags?